config/initializers/honeybadger.rb

```diff
@@ -0,0 +1,4 @@
+Honeybadger.configure do |config|
```

Comment:
I think we can just use straight env vars to config honeybadger, https://docs.honeybadger.io/lib/ruby/gem-reference/configuration.html#configuring-with-environment-variables-12-factor-style

>All configuration options can also be read from environment variables (ENV). To do this, uppercase the option name, replace all non-alphanumeric characters with underscores, and prefix with HONEYBADGER_.

https://docs.honeybadger.io/lib/ruby/gem-reference/configuration.html#configuration-options

If we don't go down that path then I'd probably just replace this one with a YAML template in github, similar to how caesar does the database template https://github.com/zooniverse/caesar/blob/e3778e0368356b2d5a40abdace37abcea53ab6df/config/database.yml#L4

See https://docs.honeybadger.io/lib/ruby/getting-started/introduction.html#how-configuration-works

Reply:
Yeah that didn't work in my testing. Setting the env vars and running a console didn't pick them up correctly. Is an initializer worse somehow than a yml template?